libs/confman/src/detail/config_dbus_client.cpp

```diff
@@ -0,0 +1,62 @@
+#include <tfc/confman/detail/config_dbus_client.hpp>
+#include <tfc/dbus/exception.hpp>
+#include <tfc/dbus/sd_bus.hpp>
+#include <tfc/dbus/sdbusplus_meta.hpp>
+#include <tfc/dbus/string_maker.hpp>
+#include <tfc/progbase.hpp>
+
+#include <fmt/format.h>
+#include <boost/asio.hpp>
+#include <sdbusplus/asio/connection.hpp>
+#include <sdbusplus/asio/object_server.hpp>
+
+namespace tfc::confman::detail {
+
+static std::string replace_all(std::string const& input, std::string_view what, std::string_view with) {
```

Comment:
Why force the caller to make a copy. If you take a & and modify input directly this function becomes more useful.

Reply:
I see why you did this.